src/ser-ext-ondemandDirective.ts

```diff
@@ -27,7 +28,7 @@ class OnDemandController implements ng.IController {
     private intervalShort: number = 3000;
     private intervalLong: number = 6000;
     private links: string[];
-    private noPropertiesSet: boolean = true;
+    private noPropertiesSet: boolean = false;
```

Comment:
typing boolean is redundant in this case this is boolean since u set this to false by default

Reply:
fixed with https://github.com/senseexcel/ser-ext-ondemand/commit/20834c08ae836aeaa95d2c8de8465dacbb87c2df